tracee-ebpf/tracee/events_config.go

```diff
@@ -91,949 +91,6015 @@ const (
-	TimerfdGettime32EventID:       {{Type: "int", Name: "ufd"}, {Type: "struct old_itimerspec32*", Name: "otmr"}},
-	TimerfdSettime32EventID:       {{Type: "int", Name: "ufd"}, {Type: "int", Name: "flags"}, {Type: "struct old_itimerspec32*", Name: "utmr"}, {Type: "struct old_itimerspec32*", Name: "otmr"}},
-	UtimensatTime32EventID:        {{Type: "unsigned int", Name: "dfd"}, {Type: "char*", Name: "filename"}, {Type: "struct old_timespec32*", Name: "t"}, {Type: "int", Name: "flags"}},
-	MqTimedsendTime32EventID:      {{Type: "mqd_t", Name: "mqdes"}, {Type: "char*", Name: "u_msg_ptr"}, {Type: "unsigned int", Name: "msg_len"}, {Type: "unsigned int", Name: "msg_prio"}, {Type: "struct old_timespec32*", Name: "u_abs_timeout"}},
-	MqTimedreceiveTime32EventID:   {{Type: "mqd_t", Name: "mqdes"}, {Type: "char*", Name: "u_msg_ptr"}, {Type: "unsigned int", Name: "msg_len"}, {Type: "unsigned int*", Name: "u_msg_prio"}, {Type: "struct old_timespec32*", Name: "u_abs_timeout"}},
-	RtSigtimedwaitTime32EventID:   {{Type: "sigset_t*", Name: "uthese"}, {Type: "siginfo_t*", Name: "uinfo"}, {Type: "struct old_timespec32*", Name: "uts"}, {Type: "size_t", Name: "sigsetsize"}},
-	FutexTime32EventID:            {{Type: "u32*", Name: "uaddr"}, {Type: "int", Name: "op"}, {Type: "u32", Name: "val"}, {Type: "struct old_timespec32*", Name: "utime"}, {Type: "u32*", Name: "uaddr2"}, {Type: "u32", Name: "val3"}},
-	SchedRrGetInterval32EventID:   {{Type: "pid_t", Name: "pid"}, {Type: "struct old_timespec32*", Name: "interval"}},
-	ClockGettime32EventID:         {{Type: "clockid_t", Name: "which_clock"}, {Type: "struct old_timespec32*", Name: "tp"}},
-	ClockSettime32EventID:         {{Type: "clockid_t", Name: "which_clock"}, {Type: "struct old_timespec32*", Name: "tp"}},
-	ClockGetresTime32EventID:      {{Type: "clockid_t", Name: "which_clock"}, {Type: "struct old_timespec32*", Name: "tp"}},
-	ClockNanosleepTime32EventID:   {{Type: "clockid_t", Name: "which_clock"}, {Type: "int", Name: "flags"}, {Type: "struct old_timespec32*", Name: "rqtp"}, {Type: "struct old_timespec32*", Name: "rmtp"}},
-	TimerGettime32EventID:         {{Type: "timer_t", Name: "timer_id"}, {Type: "struct old_itimerspec32*", Name: "setting"}},
-	TimerSettime32EventID:         {{Type: "timer_t", Name: "timer_id"}, {Type: "int", Name: "flags"}, {Type: "struct old_itimerspec32*", Name: "new"}, {Type: "struct old_itimerspec32*", Name: "old"}},
-	RecvmmsgTime32EventID:         {{Type: "int", Name: "fd"}, {Type: "struct mmsghdr*", Name: "mmsg"}, {Type: "unsigned int", Name: "vlen"}, {Type: "unsigned int", Name: "flags"}, {Type: "struct old_timespec32*", Name: "timeout"}},
-	OldstatEventID:                {{Type: "char*", Name: "filename"}, {Type: "struct __old_kernel_stat*", Name: "statbuf"}},
-	HiddenInodesEventID:           {{Type: "char*", Name: "process_hidden"}},
+var EventsConfig = map[int32]EventConfig{
```

Comment:
I think a more apt name is `EventsDefinitions` since this should define the "source of truth" and not the user-defined configuration.
Also related to my comment about essential events